erdpy/cli_shared.py

```diff
@@ -80,7 +81,8 @@ def add_wallet_args(args: List[str], sub: Any):
     sub.add_argument("--pem", required=check_if_sign_method_required(args, "--pem"), help="🔑 the PEM file, if keyfile not provided")
     sub.add_argument("--pem-index", default=0, help="🔑 the index in the PEM file (default: %(default)s)")
     sub.add_argument("--keyfile", required=check_if_sign_method_required(args, "--keyfile"), help="🔑 a JSON keyfile, if PEM not provided")
-    sub.add_argument("--passfile", required=(utils.is_arg_present(args, "--keyfile")), help="🔑 a file containing keyfile's password, if keyfile provided")
+    sub.add_argument("--passfile", help="🔑 a file containing keyfile's password, if keyfile provided")
+    sub.add_argument("--password", help="🔑 the keyfile's password, if keyfile provided")
```

Comment:
Not sure we want this (or perhaps it can be safely passed as ENV variable). Will double check with another reviewer.

Reply:
Removed too